iscsi/iscsi.go

```diff
@@ -19,7 +19,7 @@ var log *logger.Logger
 type statFunc func(string) (os.FileInfo, error)
 type globFunc func(string) ([]string, error)
 
-type secrets struct {
+type Secrets struct {
```

Comment:
Cool, I was just hiding this under the Connector parameters, is that cumbersome?

Reply:
If you don't expose Secrets, then you can't use the struct literal syntax for creation, but you're still expected to set the secrets (at least you were setting them in the example).  Exposing the secrets makes this process a bit easier since you can 1-shot create the Connector object without needing additional steps to set the secrets.

Reply:
Oh, yeah.. got ya.